### DISCUSSION TITLE
browsers: cannot open download location in file manager (dbus)

### DISCUSSION THREAD

**Discussion by githlp** [netblue30/firejail#5215] [Q&A]
When i download in browsers to the set default download folder (in my case ~/Downloads) then after download is finished and i click on "open Download destination" then nothing happens. the button used to open the dir in my default folder browse app thunar.

i did not see any issue or discussion regarding this topic, on the other hand i cannot image other people do not have this issue.. ?

**Comment by rusty-snake** [comment]
_A good report contains information like firejail version, used browser (e.g. firefox), …._

Add `dbus-user.talk org.freedesktop.FileManager1` to your browsers .local and try again.

> **Warning**: This may adds a sandbox hole which allows to create folders, copy/move files, ... escape the sandbox.
